content/tutorials/jwt-authentication.textile

```diff
@@ -136,96 +139,97 @@ This step shows you how to create the server code from scratch. If you get stuck
+
+7. In this step you will create a simple web server using the "Express web framework":https://www.npmjs.com/package/express to serve static files. In a later step you will also use this server to create an API route for clients to authenticate with.
 
 ```[javascript]
-const jwtPayload = {                                                                                                                                                                                                                                           
-    'x-ably-capability': JSON.stringify({ '*': ['publish', 'subscribe'] }),                                                                                                                                                                                    
-    'x-ably-clientId': 'OPTIONAL_CLIENT_ID'                                                                                                                                                                                                                    
-}                                                                                                                                                                                                                                                              
-                                                                                                                                                                                                                                                               
-const jwtOptions = {                                                                                                                                                                                                                                           
-    expiresIn: ttlSeconds,                                                                                                                                                                                                                                     
-    keyid: `${keyName}`                                                                                                                                                                                                                                         
-}      
-```
+const express = require('express');
 
-9. To finish the web app you use "Express":https://www.npmjs.com/package/express to build a simple server to serve files on your localhost:
+const app = express();
 
-```[javascript]
-const express = require('express'),
-    app = express();
 app.use('/', express.static(__dirname))
 ```
 
 The Express server uses a middleware to serve the content, and uses @express.static(__dirname)@ to specify that any HTML files in the root folder are served directly.
```

Comment:
"Middleware" refers to a collection of middleware functions. So "a middleware _function_" is the correct way to refer to one of them.